org/mozilla/jss/tests/all.pl

```diff
@@ -131,7 +132,6 @@ sub setup_vars {
         $scriptext = "sh";
     }
 
-    $ENV{CLASSPATH}  = "";
```

Comment:
Since we have `$classpath` and explicitly pass it on all `java` calls, this variable can disappear.